spec/support/features/doc_capture_helper.rb

```diff
@@ -37,24 +37,14 @@ def complete_doc_capture_steps_before_document_capture_step(user = user_with_2fa
   end
 
   def complete_doc_capture_steps_before_capture_complete_step(user = user_with_2fa)
-    if FeatureManagement.document_capture_step_enabled?
-      complete_doc_capture_steps_before_document_capture_step
-      attach_and_submit_images
-    else
-      complete_doc_capture_steps_before_mobile_back_image_step(user)
-      attach_image
-      click_idv_continue
-    end
+    complete_doc_capture_steps_before_document_capture_step
+    attach_and_submit_images
   end
 
   def idv_capture_doc_mobile_front_image_step
     idv_capture_doc_step_path(step: :mobile_front_image)
   end
```

Comment:
Should be removed?

```suggestion
```

There's also at least one reference to this function above in `complete_doc_capture_steps_before_mobile_back_image_step`.

Reply:
Yep